pkg/ecdsa/dkg/member.go

```diff
@@ -111,24 +126,199 @@ func (skgm *symmetricKeyGeneratingMember) MarkInactiveMembers(
 	filter.FlushInactiveMembers()
 }
 
+// initializeTssRoundOne returns a member to perform next protocol operations.
+func (skgm *symmetricKeyGeneratingMember) initializeTssRoundOne() *tssRoundOneMember {
+	tssPartyID, groupTssPartiesIDs := generateTssPartiesIDs(
+		skgm.id,
+		skgm.group.MemberIDs(),
```

Comment:
Not in this PR but I would consider renaming `MemberIDs()` to `MemberIndexes()`, `InactiveMemberIDs()` to `InactiveMemberIndexes()` and so on.

`skgm.group.MemberIDs()` is just an array of numbers from 1 to 100 and it makes sense to generate TSS party ID based on them.

For a moment I was confused and thought `skgm.group.MemberIDs()` is a list of identifiers, as outputted by the Sortition Pool, especially that the documentation of `MemberIDs()` says:

> MemberIDs returns IDs of all group members, as initially selected to the group

This would not make sense because we would have TSS party IDs duplicated.

Reply:
Left a TODO: https://github.com/keep-network/keep-core/pull/3112/commits/98929caf210936fcdf43b39e12c7158a050f38f4

Reply:
Done in https://github.com/keep-network/keep-core/pull/3428